test/Microsoft.Dnx.Testing.CommonUtils/DnxSdk/DnuPackOutput.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Runtime.Versioning;
+using NuGet;
+
+namespace Microsoft.Dnx.Testing
+{
+    public class DnuPackOutput : ExecResult
+    {
+        public DnuPackOutput(string outputPath, string packageName, string configuration)
+        {
+            RootPath = outputPath;
+            PackageName = packageName;
+            Configuration = configuration;
+            var basePath = Path.Combine(RootPath, Configuration);
+            PackagePath = Directory.EnumerateFiles(basePath, $"*{Constants.PackageExtension}")
```

Comment:
You don't need to enumerate the files. I'm not sure why we did that. The output is predictable (unless we change it..)